.devcontainer/Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG VARIANT=3.8
+FROM --platform=linux/amd64 mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
+
+# System update/upgrade
+RUN apt-get update
+RUN apt-get -y upgrade
+
+# Install dependencies
+RUN apt-get install -y libxml2-dev libxslt-dev libxslt1-dev unixodbc-dev libpq-dev libssl-dev libffi-dev libcurl4-openssl-dev libblas-dev libatlas-base-dev libsasl2-dev
+RUN apt-get install -y make python3-pip python3-dev
+
+# Docker
+RUN apt-get update \
```

Comment:
I thin this step is not necessary again, because we previously already run it on  line 5.